### ISSUE TITLE
CD's CDBase declaration is mandatory

### ISSUE THREAD

**Issue by kohlhase** [OpenMath/OMSTD#22]
see https://github.com/OpenMath/OM3/issues/138

**Comment by davidcarlisle** [comment]
Certainly we shouldn't make cdbase mandatory, we could add some words that say openmath.org/cd could be taken as a default if needed, I thought we had some words like that but  perhaps it was in mathml3, I'll propose something

**Comment by kohlhase** [comment]
Apparently, the abstract CD specification and `omcd2.rnc` say different things about which fields are mandatory. Here is a list  
*  mandatory in OM2 and optional in RNC
  * `cdbase`
  * `ReviewDate` 
  * `Description`
  * `CDReviewdate`
* OM2 mandates a CD base, whereas the RNC has an optional `CDBase` or `CDURL`. 
* In OM2  a "revision date" is mandatory and in the RNC a 'CDRevision' is a number. But maybe `CDDate` is the revision date. 

The contents of `CDDefinition` seem to macth between OM2 and RNC.

**Comment by kohlhase** [comment]
Now we have the choice in what to do with these inconsistencies. @lars-hellstrom argues that the RNC is more well-thought-out than OM2. 
I wonder what the data (i.e. the CDs we have) say.

**Comment by davidcarlisle** [comment]
I think confirming cdbase is optional is the only possibility really. It's
compatible with existing schema and every reference to cdbase in abstract
openmath objects, and openmath xml says cdbase is optional so saying it's
mandatory in an abstract cdbase looks like an oversight, however I have no
problem with saying abstractly it always has a cdbase, being openmath.org/cd
if not specified otherwise.

**Comment by kohlhase** [comment]
> however I have no problem with saying abstractly it always has a cdbase, being openmath.org/cd if not specified otherwise.

I agree that is the way forward for cdbase. But there are more inconsistencies we need to address.
